main.tf

```diff
@@ -7,7 +7,7 @@ resource "kubernetes_namespace" "gatekeeper" {
       "cloud-platform.justice.gov.uk/is-production"    = var.is_production
       "cloud-platform.justice.gov.uk/environment-name" = var.environment_name
       "admission.gatekeeper.sh/ignore"                 = "no-self-managing"
-      "pod-security.kubernetes.io/audit"               = "privileged"
+      "pod-security.kubernetes.io/enforce"               = "privileged"
```

Comment:
Even the Gatekeeper Admission webhook ignore this namepsace, would it not be needed in order for the podSecurity to enforce privileged permissions when deploying the gatekeeper itself? I think when we start doing bypassing the psp, it will help ensure the psa is set as privileged (even though the default is privileged)